package.json

```diff
@@ -123,16 +119,10 @@
     "whatwg-fetch": "^2.0.3"
   },
   "scripts": {
-    "build": "npm run gulp:prod:build && npm run webpack:prod:build",
-    "dist": "NODE_ENV=production npm run build",
-    "watch": "npm-run-all --parallel gulp:dev:watch webpack:dev:watch",
-    "start": "npm run watch",
-    "gulp:dev:watch": "gulp watch",
-    "gulp:prod:build": "gulp build",
-    "webpack:dev:watch": "webpack --config ./client/webpack.config.js --mode development --progress --watch",
-    "webpack:prod:build": "webpack --config ./client/webpack.config.js --mode production",
-    "fix:js": "eslint --fix ./",
-    "lint:js": "eslint --report-unused-disable-directives ./",
+    "start": "webpack --config ./client/webpack.config.js --mode development --progress --watch",
+    "build": "webpack --config ./client/webpack.config.js --mode production",
+    "fix:js": "eslint --ext .js,.ts,.tsx --fix ./",
+    "lint:js": "eslint --ext .js,.ts,.tsx --report-unused-disable-directives --max-warnings 16 ./client",
```

Comment:
It would be nice if we didn't have to declare the ext here. Is that a performance decision? Is there any way to do this in the configuration? 
Do we currently lint JS in html files ( I don't think so but checking)

Reply:
Do we need a script to clear any already built files? I know that sometimes this gets people stuck - me included.

Reply:
Re `--ext`, I believe this is [needed](https://eslint.org/docs/user-guide/command-line-interface#--ext) so `ESLint` will pick those files up when used with a folder as the target (`./`). See also https://typescript-eslint.io/docs/linting/#running-eslint.

I don’t think there is a way to do this via the configuration. `overrides` could help with this as mentioned in the docs, but I don’t think they’re intended to be used _just_ for the purpose of specifying which files to lint.

---

Re clearing built files, I’d be keen for us to try Webpack 5’s `output.clean: true` setting, and-or completely change how we output files so we no longer have this mess of 20 different output folders – but I’d rather do this in a follow-up PR.

Reply:
All makes sense, thanks for clarifying.